src/liedle/dictionary.js

```diff
@@ -0,0 +1,5758 @@
+    "bibbs",
+    "antes",
+    "lulus",
+    "gnarl",
+    "zippy",
+    "ivied",
+    "epees",
+    "wimps",
+    "tromp",
+    "grail",
+    "yoyos",
+    "poufs",
+    "hales",
+    "roust",
+    "cabal",
+    "rawer",
+    "pampa",
+    "mosey",
+    "kefir",
+    "burgs",
+    "unmet",
+    "cuspy",
+    "boobs",
+    "boons",
+    "hypes",
+    "dynes",
+    "nards",
+    "lanai",
+    "yogis",
+    "sepal",
+    "quark",
+    "toked",
+    "prate",
+    "ayins",
+    "hawed",
+    "swigs",
+    "vitas",
+    "toker",
+    "doper",
+    "bossa",
+    "linty",
+    "foist",
+    "mondo",
+    "stash",
+    "kayos",
+    "twerp",
+    "zesty",
+    "capon",
+    "wimpy",
+    "rewed",
+    "fungo",
+    "tarot",
+    "frosh",
+    "kabob",
+    "pinko",
+    "redid",
+    "mimeo",
+    "heist",
+    "tarps",
+    "lamas",
+    "sutra",
+    "dinar",
+    "whams",
+    "busty",
+    "spays",
+    "mambo",
+    "nabob",
+    "preps",
+    "odour",
+    "cabby",
+    "conks",
+    "sluff",
+    "dados",
+    "houri",
+    "swart",
+    "balms",
+    "gutsy",
+    "faxed",
+    "egads",
+    "pushy",
+    "retry",
+    "agora",
+    "drubs",
+    "daffy",
+    "chits",
+    "mufti",
+    "karma",
+    "lotto",
+    "toffs",
+    "burps",
+    "deuce",
+    "zings",
+    "kappa",
+    "clads",
+    "doggy",
+    "duper",
+    "scams",
+    "ogler",
+    "mimes",
+    "throe",
+    "zetas",
+    "waled",
+    "promo",
+    "blats",
+    "muffs",
+    "oinks",
+    "viand",
+    "coset",
+    "finks",
+    "faddy",
+    "minis",
+    "snafu",
+    "sauna",
+    "usury",
+    "muxes",
+    "craws",
+    "stats",
+    "condo",
+    "coxes",
+    "loopy",
+    "dorms",
+    "ascot",
+    "dippy",
+    "execs",
+    "dopey",
+    "envoi",
+    "umpty",
+    "gismo",
+    "fazed",
+    "strop",
+    "jives",
+    "slims",
+    "batik",
+    "pings",
+    "sonly",
+    "leggo",
+    "pekoe",
+    "prawn",
+    "luaus",
+    "campy",
+    "oodle",
+    "prexy",
+    "proms",
+    "touts",
+    "ogles",
+    "tweet",
+    "toady",
+    "naiad",
+    "hider",
+    "nuked",
+    "fatso",
+    "sluts",
+    "obits",
+    "narcs",
+    "tyros",
+    "delis",
+    "wooer",
+    "hyped",
+    "poset",
+    "byway",
+    "texas",
+    "scrod",
+    "avows",
+    "futon",
+    "torte",
+    "tuple",
+    "carom",
+    "kebab",
+    "tamps",
+    "jilts",
+    "duals",
+    "artsy",
+    "repro",
+    "modem",
+    "toped",
+    "psych",
+    "sicko",
+    "klutz",
+    "tarns",
+    "coxed",
+    "drays",
+    "cloys",
+    "anded",
+    "piker",
+    "aimer",
+    "suras",
+    "limos",
+    "flack",
+    "hapax",
+    "dutch",
+    "mucky",
+    "shire",
+    "klieg",
+    "staph",
+    "layup",
+    "tokes",
+    "axing",
+    "toper",
+    "duvet",
+    "cowry",
+    "profs",
+    "blahs",
+    "addle",
+    "sudsy",
+    "batty",
+    "coifs",
+    "suety",
+    "gabby",
+    "hafta",
+    "pitas",
+    "gouda",
+    "deice",
+    "taupe",
+    "topes",
```

Comment:
Starting from about line 2000 onwards the words start to get less and less commonly known and from about line 3000 onwards an average person probably won't know that most of these words exist. The word list might not be very suitable for Liedle.

Reply:
One of the things I like about playing Wordle/Liedle is learning new words. Sure, it's frustrating when the answer is something bizarre, but I think it adds to the fun/luck component of the game. Maybe in the future, there could be two dictionaries - one to act as a pool of secret words and one to refer to when determining when a guess is a valid word.